x-pack/plugins/security_solution/public/detections/components/rules/query_bar/index.tsx

```diff
@@ -253,7 +253,10 @@ export const QueryBarDefineRule = ({
     }
   };
 
-  const actionTimelineToHide = useMemo<ActionTimelineToShow[]>(() => ['duplicate'], []);
+  const actionTimelineToHide = useMemo<ActionTimelineToShow[]>(
```

Comment:
This seems doesn't change, please consider put this as a constant and move out of the component.
```const actionTimelineToHide = ['duplicate', 'createFrom'];
```